pkg/credentials/editor_test.go

```diff
@@ -0,0 +1,74 @@
+package credentials
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+type Editor struct {
+	configPath string
+}
+
+func createEditor(editor Editor) *ConfigEditor {
+	credentialsFile := "credentials.yml.enc"
+	masterKeyFile := "master.key"
+	configEditor := NewConfigEditor(
+		editor.configPath,
+		credentialsFile,
+		masterKeyFile,
+		"vim",
+		executor,
+		fileManager,
+	)
+	return configEditor
+}
+
+func TestOpenEditor(t *testing.T) {
+	t.Run("there is no credentials", func(t *testing.T) {
+
+		t.Run("master.key exists", func(t *testing.T) {
+			// TODO check master.key only to see error
+		})
+
+		t.Run("master.key does not exist", func(t *testing.T) {
+			// run openEditor
+			editor := createEditor(Editor{configPath: "./config"})
+			err := editor.Open()
+			require.NoError(t, err)
+
+			require.FileExistsf(t, "config/credentials.yml.enc", "this file does not exist")
```

Comment:
do we need 'f' version of FileExists?

Reply:
BTW, it's a bit confusing to read expectation, and message error on the same line. The error message is in different meaning from expectation though.